ProcessLib/NumericsConfig.h

```diff
@@ -129,4 +129,7 @@ static_assert(std::is_same<detail::GlobalMatrixType::IndexType,
 /// GlobalMatrixType::IndexType and the GlobalVectorType::IndexType.
 using GlobalIndexType = detail::GlobalMatrixType::IndexType;
 
+/// A type for scalar number
+using ScalarType = detail::GlobalMatrixType::ScalarType;
```

Comment:
It is for the newly added member of matrix class,  void getValues(PetscScalar u[]) of  void getValues(double u[]). It is used to avoid converting float type of every entries of the solution vector.